README.md

```diff
@@ -84,8 +91,8 @@ Then, open `.env.local` and add the missing environment variables:
   sure to verify the email in the AWS SES dashboard.
 - `NEXT_PUBLIC_FACEBOOK_ID` - Facebook APP ID. You can get it
   [here](https://developers.facebook.com/docs/development/create-an-app).
-- `AUTH0_AUDIENCE` - The url for Auth0 audience. Normally has the following structure `https://your-tenant.auth0.com/api/v2/`
-
+- `AUTH0_AUDIENCE` - The url for Auth0 audience. Normally has the following
+  structure `https://your-tenant.auth0.com/api/v2/`
```

Comment:
Seeing some formatting changes. Are we using the same prettier config?

Reply:
We should be but I might same some conflicts :disappointed: Thanks for letting me know of this :+1:

Reply:
:+1: